xmltodict.py

```diff
@@ -14,7 +14,7 @@
 from inspect import isgenerator
 
 __author__ = 'Martin Blech'
-__version__ = "0.14.2"
+__version__ = "0.14.3"
```

Comment:
Please revert the version bump, I'll do that in a separate commit before releasing.